src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -104,7 +106,24 @@ export const TradeInput = memo(() => {
   const totalNetworkFeeFiatPrecision = useAppSelector(selectTotalNetworkFeeUserCurrencyPrecision)
   const manualReceiveAddressIsValidating = useAppSelector(selectManualReceiveAddressIsValidating)
   const sellAmountCryptoPrecision = useAppSelector(selectSellAmountCryptoPrecision)
-  const slippagePercentage = useAppSelector(selectTradeSlippagePercentageDecimal)
+  const sellAmountBeforeFeesUserCurrency = useAppSelector(selectSellAmountUserCurrency)
+  const slippageDecimal = useAppSelector(selectTradeSlippagePercentageDecimal)
+
+  const priceImpactPercentage = useMemo(() => {
+    if (!sellAmountBeforeFeesUserCurrency || !buyAmountAfterFeesUserCurrency) return bn('0')
+
+    const tradeDifference = bn(sellAmountBeforeFeesUserCurrency)
+      .minus(buyAmountAfterFeesUserCurrency)
```

Comment:
As with the previous PR, it feels kind of strange to be using the after fees amount here, as they shouldn't impact the messaging we give users related to pool liquidity.

Reply:
https://github.com/shapeshift/web/pull/5033/commits/40e7adaa2388993eff19afbb4887fbc26644e98c

Reply:
https://github.com/shapeshift/web/pull/5033/commits/ae775300ec56af378f0225a167859f62ea050e32